site/layouts/shortcodes/callout-dark-variants.html

```diff
@@ -0,0 +1,9 @@
+{{- /* Outputs badge to identify the first version something was added */ -}}
```

Comment:
This comment must be modified to reflect the content of this file. Same thing for `site/layouts/shortcodes/deprecated-in.html`.